dotnet/src/SemanticKernel.Core/Planning/InstrumentedPlan.cs

```diff
@@ -70,7 +83,7 @@ public FunctionView Describe()
 
     #region private ================================================================================
 
-    private readonly IPlan _plan;
+    private readonly Plan _plan;
```

Comment:
Same comment as above, using `ISKFunction` instead of `Plan` should make this class more testable.